extensions/apiconnect/src/__tests__/acceptance/apiconnect.acceptance.ts

```diff
@@ -42,6 +42,28 @@ describe('Extension for IBM API Connect - OASEnhancer', () => {
     expect(spec).to.containDeep(EXPECTED_SPEC);
   });
 
+  it('adds x-ibm-name to apiSpec info', async () => {
+    app.setMetadata({name: 'lb4-app', version: '1.0.0', description: ''});
+    const EXPECTED_SPEC = {
+      info: {
+        'x-ibm-name': 'lb4-app',
+      },
+    };
+    const spec = await server.getApiSpec();
+    expect(spec).to.containDeep(EXPECTED_SPEC);
+  });
+
+  it('adds default x-ibm-name to apiSpec info', async () => {
+    app.unbind(CoreBindings.APPLICATION_METADATA);
+    const EXPECTED_SPEC = {
+      info: {
+        'x-ibm-name': 'LoopBack Application',
```

Comment:
can we get the default value for x-ibm-name from the app's package.json

Reply:
See above.